dev/test/bundle.ts

```diff
@@ -37,17 +41,19 @@ export function verifyMetadata(
   expectEmptyContent = false
 ): void {
   if (!expectEmptyContent) {
-    expect(meta.totalBytes).greaterThan(0);
+    expect(parseInt(meta.totalBytes!.toString())).greaterThan(0);
   } else {
-    expect(meta.totalBytes).to.equal(0);
+    expect(parseInt(meta.totalBytes!.toString())).to.equal(0);
   }
   expect(meta.id).to.equal(TEST_BUNDLE_ID);
   expect(meta.version).to.equal(TEST_BUNDLE_VERSION);
   expect(meta.totalDocuments).to.equal(totalDocuments);
-  expect(meta.createTime).to.deep.equal(createTime);
+  expect(google.protobuf.Timestamp.fromObject(meta.createTime!)).to.deep.equal(
```

Comment:
Fix for this: https://gist.github.com/schmidt-sebastian/3f53054d1e1c2d8238e473bfc133a1de

Reply:
Done.